openstudiocore/src/utilities/core/test/ApplicationPathHelpers_GTest.cpp

```diff
@@ -51,3 +113,204 @@ TEST(ApplicationPathHelpers, Strings)
   EXPECT_TRUE(exists(energyplusDirectory));
   EXPECT_TRUE(exists(energyplusExecutable));
 }
+
+
+
+// This is really in PathHelpers but used by ApplicationPathHelpers
+TEST(ApplicationPathHelpers, findInSystemPath) {
+
+  openstudio::path dummy_dir = getApplicationBuildDirectory() / toPath("Testing/Temporary");
+  openstudio::filesystem::create_directories(dummy_dir);
+  openstudio::path dummy_file_path = dummy_dir / "dummy_file";
+
+  boost::filesystem::ofstream outfile(dummy_file_path);
+  outfile.close();
+
+  // This should fail
+  openstudio::path absolute_path_to_dummy = findInSystemPath(dummy_file_path.filename());
+  EXPECT_FALSE(exists(absolute_path_to_dummy));
+
+  // Append the containing dir to the PATH
+  auto current_path = std::getenv("PATH");
+  std::string new_path = toString(dummy_dir) + getPlatformDelimiterForPath() + current_path;
```

Comment:
@jmarrec would `pathDelimiter()` be ok?  `getPlatformDelimiterForPath` is a bit lengthy

Reply:
Nevermind I see this is just a local function

Reply:
I had posted a comment suggesting that I move it in PathHelpers.cpp above (https://github.com/NREL/OpenStudio/pull/3497jmarrec:PR_opened/3158_SymlinkPathProblems). I ended up doing this, and calling it pathDelimiter as you suggest